res/controllers/midi-components-0.0.js

```diff
@@ -583,11 +594,16 @@
         };
         this.EffectUnitKnob.prototype = new Pot({
             group: this.group,
+            input: function (channel, control, value, status, group) {
```

Comment:
@MelGrubb if you want to try this with a master build from the build server before #1218 is merged you'll need to comment out this function.